labgrid/remote/client.py

```diff
@@ -925,7 +925,7 @@ def create():
 
 def find_role_by_place(config, place):
     for role, role_config in config.items():
-        resources, _ = target_factory.normalize_config(role_config)
+        resources, _ = target_factory.__class__.normalize_config(role_config)
```

Comment:
The __class__ should be unnecessary here. You can use staticmethods on instances as well.